ios/FluentUI.Demo/FluentUI.Demo/Demos/TypographyTokensDemoController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class TypographyTokensDemoController: DemoTableViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: Constants.cellID)
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return AliasTokens.TypographyTokens.allCases.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: Constants.cellID, for: indexPath)
+        let typographyToken = AliasTokens.TypographyTokens.allCases[indexPath.row]
+
+        cell.selectionStyle = .none
+
+        var contentConfiguration = cell.defaultContentConfiguration()
+        let text = "\(typographyToken.text)"
+        contentConfiguration.attributedText = NSAttributedString(string: text,
+                                                                 attributes: [
+                                                                    .font: UIFont.fluent(aliasTokens.typography[typographyToken])
+                                                                 ])
+        contentConfiguration.textProperties.alignment = .center
+        cell.contentConfiguration = contentConfiguration
+
+        return cell
+    }
+
+    private var aliasTokens: AliasTokens {
+        guard let fluentTheme = self.view.window?.fluentTheme else {
+            return AliasTokens()
+        }
+        return fluentTheme.aliasTokens
+    }
+
+    private struct Constants {
+        static let cellID: String = "cellID"
```

Comment:
You should be able to use `TableViewCell.identifier` instead instead of storing this in this struct.

Reply:
Ah I technically could, but I chose not to because I'm not using the Fluent `TableViewCell` here, and I'd rather not mix and match the two.

(FWIW this class is an almost perfect copy of the `ColorTokensDemoController`, which uses the same paradigm.)